client/src/locales/en.json

```diff
@@ -216,6 +216,10 @@
   "dashboard.index.no-arch-sacks": "There are no archived sacks yet.",
   "dashboard.index.no-sacks": "There are no sacks yet.",
   "dashboard.index.show-arch-sacks": "Show archived sacks",
+  "delete-link.has-expired": "Delete account link has expired. If you still want to delete account go to {link} page.",
+  "delete-link.has-expired-link": "user profile page",
```

Comment:
If this one is used in previous text then you'll get double `page` word at the end of the sentence.